doc/architecture/squid.md

```diff
@@ -723,27 +719,20 @@ ocean.agreements
 
 ---
 
-ocean.agreements.conditions.payment
+ocean.agreements.conditions
```

Comment:
I don't totally understand that. Why do we have a module that has `lockReward`, `grantAccess`...?
This behaviour is part of the different conditions/templates, right? They are like shortcuts to work with `EscrowAccessSecretStoreTemplate` template?

Reply:
Yes, these form part of the `Access` agreement flow.